target/linux/realtek/files-5.10/drivers/net/dsa/rtl83xx/common.c

```diff
@@ -424,24 +424,30 @@ static int __init rtl83xx_get_l2aging(struct rtl838x_switch_priv *priv)
 	return t;
 }
 
-/* Caller must hold priv->reg_mutex */
+/*
+ * Caller must hold priv->reg_mutex
+ */
 int rtl83xx_lag_add(struct dsa_switch *ds, int group, int port, struct netdev_lag_upper_info *info)
 {
 	struct rtl838x_switch_priv *priv = ds->priv;
 	int i;
 	u32 algomsk = 0;
 	u32 algoidx = 0;
+
+	pr_info("%s: Adding port %d to LA group %d\n", __func__, port, group);
```

Comment:
There's a pr_debug at the end of the function. These drivers are verbose enough as they are, I don't think this line really adds value, being so verbose all the time.